extensions/amp-lightbox-gallery/amp-lightbox-gallery.md

```diff
@@ -37,9 +37,9 @@ limitations under the License.
 </table>
 
 ## Behavior
-All you need to do is to include the 'lightbox' attribute on an `<amp-img>` or `<amp-carousel>`. A typical usage looks like this:
+All you need to do is to include the extension script and put the 'lightbox' attribute on an `<amp-img>` or `<amp-carousel>`. A typical usage looks like this:
 
-### Lightbox with AMP-IMG
+### Lightbox with `<amp-img>`
 
 ```html
 <amp-img src="image1" width=200 height=100 lightbox></amp-img>
```

Comment:
In code samples, attribute values are enclosed in double quotes (e.g., `"width="200"`). Please apply that standard to your code sample here and below.